### DISCUSSION TITLE
Inquiry About EJBCA Roadmap/Timeline

### DISCUSSION THREAD

**Discussion by oeztolga** [Keyfactor/ejbca-ce#744] [Q&A]
Dear EJBCA Community,

My team and I have been utilising EJBCA and are highly appreciative of its value to our projects. However, we are curious if there is any publicly available information about the roadmap or future plans for EJBCA.

Gaining an understanding of the project's development direction would be extremely beneficial for planning purposes. If anyone can provide insights or direct me to the relevant resource, I would be most grateful.

Best regards

**Comment by primetomas** [comment]
You will find some stuff [here](https://www.ejbca.org/resources/). 
The Community project does not commit to long term roadmaps in a public document, [EJBCA Enterprise](https://www.keyfactor.com/products/ejbca-enterprise/) of course does. We do have yearly events like the [Community Meetup](https://www.keyfactor.com/events/community-tech-meetup-24/) where roadmaps are discussed in a roadmap session. Reflections on the last event is [here](https://www.keyfactor.com/blog/reflections-from-this-years-keyfactor-community-tech-meetup/).
We also have a [Youtube channel](https://www.youtube.com/KeyfactorCommunity) where such things can come up.
